controllers/lists_controller.go

```diff
@@ -36,6 +36,11 @@ type ListJsonFormat struct {
 	ListOptionId int64
 }
 
+type AllListJSONFormat struct {
```

Comment:
__[golint]__
* [golint] exported type AllListJSONFormat should have comment or be unexported